docs/Artefacts/offramps.md

```diff
@@ -1002,7 +1004,7 @@ Example:
       dir: "temp/kv" # directory to store data in
 ```
 
-One response event is sent for every command in the handled event. On success to the `out` port, in the error case (invalid format or error during kv operation) to the `err` port.
+One response event is sent for every command in the handled event. On success to the `out` port, in the error case (malformed or error during kv operation) to the `err` port.
```

Comment:
I think you meant malformed *format* here.

Reply:
I changed it to `malformed data` which seems to fit best :)